gdal/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py

```diff
@@ -354,6 +354,7 @@ def get_bands(filename_or_ds: PathOrDS, band_nums: Optional[MaybeSequence[int]]
         if band is None:
             raise Exception(f'Could not get band {band_num} from file {filename_or_ds}')
         if ovr_idx:
+            ovr_idx = get_ovr_idx(ds, ovr_idx)
```

Comment:
Reviewing what get_over_idx() does and its use just below, I realize it uses a different convention than the GDAL API itself, with 0 meaning the full resolution image, 1 the first overview level, etc. The doc of get_ovr_idx() should be enhanced to underline that.  And the doc of this method too.

Reply:
Good point. thanks, done.